### PR TITLE
Added the Terminologies file

### PR DESCRIPTION
**Pull request by VenkyOS**
Added the Terminologies file. Please review the same and approve the PR.

Signed-off-by: Venkataraman Padmanabhan <venkataraman.padmanabhan@seagate.com>

### CODE REVIEW

File: doc/Terminologies.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Terminologies and Definitions
```
Comment (huanghua78) [inline doc/Terminologies.rst:1]: Please list as more as possible terminologies, and we can delete them if they are not needed.

Comment (VenkyOS) [reply doc/Terminologies.rst:1]: Sure

**Comment by huanghua78** [comment]
doc/PDF/Glossary.pdf

This is a good source for this task.

**Comment by huanghua78** [comment]
Hello @VenkyOS ,
Please check if this PR is still valid and needed.
If not, please close it.
If yes, please update the status.

**Pull request closed without merging**